zone/raids.cpp

```diff
@@ -1634,7 +1666,7 @@ void Raid::SetRaidDetails()
 
 void Raid::GetRaidDetails()
 {
-	std::string query = StringFormat("SELECT locked, loottype, motd FROM raid_details WHERE raidid = %lu",
+	std::string query = StringFormat("SELECT locked, loottype, motd, MarkedNPC1, MarkedNPC2, MarkedNPC3 FROM raid_details WHERE raidid = %lu",
```

Comment:
Columns need to be `snake_case` please see https://docs.eqemu.io/developer/conventions-guidelines/#column-naming

Reply:
Updated